modules/end-user-guide/pages/user-onboarding.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: CONCEPT
+:description: User onboarding
+:keywords: getting-started, user-onboarding, new-user, new-users
+:navtitle: User onboarding
+// :page-aliases:
+
+[id="user-onboarding_{context}"]
+= User onboarding
+
+If your organization is already running a {prod-short} instance, you can get started as a new user of {prod-short} by first learning how to start a new workspace and how to manage your workspaces:
+
+. xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[]
+. xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[]
+. xref:basic-actions-you-can-perform-on-a-workspace.adoc[]
+
+Beyond this, your organization has its own setup and onboarding needs regarding:
```

Comment:
@rolfe, in response to your feedback in https://github.com/eclipse-che/che-docs/pull/2252#discussion_r837674038, I rewrote that as follows:

![image](https://user-images.githubusercontent.com/78564304/161955017-7966b970-5c61-478f-883c-dcae3b9a9823.png)